api/test/e2e/admin-regions/admin-regions-get-trees.spec.ts

```diff
@@ -36,6 +38,16 @@ describe('AdminRegions - Get trees', () => {
     );
     await app.init();
 
+    await request(app.getHttpServer())
```

Comment:
Extract this into a separate function, so you don't have to repeat the same block of code in every file like you did.
Also: instead of using the sign-up endpoint, manually inject a user into the database, as it's easier to extend